### PR TITLE
[#160] Add flow-api-internal config values

### PR DESCRIPTION
**Pull request by tangrammer**
Relates #160 

... not sure if we prefer a dot or a slash character in the key name 🤔

### CODE REVIEW

File: k8s/config/dev.yaml [hunk 1]
```diff
@@ -7,4 +7,5 @@ data:
   keycloak.url: https://kc.akvotest.org/auth
   email.host: mail.akvo.org
   flow.api.root: https://api.akvotest.org/flow
+  flow.api_internal.root: https://api-internal.akvotest.org/flow
```
Comment (dlebrero) [inline k8s/config/dev.yaml]: Url should be http://flow-api-internal

Use dots.

Given that the value is the same in test and prod, does it need to be configured? 

No other service uses this configuration file.

Comment (tangrammer) [reply k8s/config/dev.yaml]: why don't we have test an prod values for `flow.api_internal ` but we do for `flow.api`?

Comment (tangrammer) [reply k8s/config/dev.yaml]: `does it need to be configured?` do we want to manage this value at a higher/more-generic level?

**Comment by tangrammer** [comment]
Closing after agreeing that this value should not usually be changed nor depends on `prod` or `env` settings

**Pull request closed without merging**